### DISCUSSION TITLE
游戏无法启动以及卸载

### DISCUSSION THREAD

**Discussion by YoraYosaki** [BeyondDimension/SteamTools#1322] [Q&A] [answered]
在没有安装游戏的前提下，修改了游戏的成就。随后在steam界面，原本‘’安装游戏‘’的标识变成了‘’启动游戏‘’的标识，但点‘’启动游戏‘’后游戏无法正常启动。在属性里面效验文件完整性为完整，文件大小为0B。随后点击‘’卸载游戏‘’无反应。在steam++上面点击下载游戏也无反应。请问一下这种情况应该怎么解决，谢谢。

**Comment by AigioL** [comment, marked as answer]
重启 Steam 能解决么

**Reply by YoraYosaki** [reply]
我试了一下可以的，谢谢。
